index.js

```diff
@@ -1,16 +1,20 @@
 import fp from 'fastify-plugin'
 import onFinished from 'on-finished'
 
+import { RedisStore } from './lib/redisStore.js'
 import { DEFAULT_OPTIONS, HEADERS } from './lib/constants.js'
 import { calculateDelay, convertToMs } from './lib/helpers.js'
 import { Store } from './lib/store.js'
 
 const slowDownPlugin = async (fastify, settings) => {
   const options = { ...DEFAULT_OPTIONS, ...settings }
-  const store = new Store(
-    convertToMs(options.timeWindow),
-    options.inMemoryCacheSize
-  )
+  const store = settings.redis
+    ? new RedisStore(
+        settings.redis,
+        'fastify-slow-down',
+        convertToMs(options.timeWindow)
+      )
+    : new Store(convertToMs(options.timeWindow), options.inMemoryCacheSize)
```

Comment:
As now we have `RedisStore` should we rename the plain `Store` in something like `LocalStore`?